web/src/components/storage/VolumeForm.jsx

```diff
@@ -722,17 +762,33 @@ export default function VolumeForm({ id, volume: currentVolume, templates = [],
 
   const { fsType, snapshots } = state.formData;
 
+  const ShowMountPointSelector = () => (
+    <MountPointFormSelect
+        id="mountPoint"
+        value={state.formData.mountPoint}
+        onChange={changeVolume}
+        volumes={currentVolume ? [currentVolume] : templates}
+        isDisabled={currentVolume !== undefined}
```

Comment:
Don't know what happened, but looks like commit 827a1d8d18687f03b3b24dadcd6cce52573a26d9 re-introduced a not expected prop that was already removed in 679b5b5558499fc422412719e302e90f19d505b7.

Do you mind having a look, please?

Reply:
@dgdavid sir, when isDisabled prop is removed some of the test cases are failing. Should we just modify the testcases there or do you think something might be wrong.

Reply:
Hello sir, can you please suggest to me what to work on next and suggest some technologies that I can learn simultaneously? Currently, I am learning Kubernetes and am confused about whether to learn Rust or Golang.

Thank you so much in advance.

Reply:
> Hello sir, can you please suggest to me what to work on next 

Sure, we can look for something that you can address with less pain that this one :wink:  But first, I think it is better to finish this one. Changelog is missing :)

> and suggest some technologies that I can learn simultaneously? Currently, I am learning Kubernetes and am confused about whether to learn Rust or Golang.
> 

Not sure what to say here :sweat_smile: I may be a bit biased by my own preferences. However, I have a non-technical suggestion: don't open multiple front lines at the same time. It's not about specializing only in one thing, but to avoid covering too much at once and having a rough understanding of nothing. IMHO, it is better to invest the required time in one technology to understand it well enough for being able to deliver good quality work. Particularly at the beginning, to lay a solid foundation. 

I know I didn't give you a path to follow, but that was on purpose :stuck_out_tongue_winking_eye:  Hope you don't mind :)

Reply:
> > Hello sir, can you please suggest to me what to work on next
> 
> Sure, we can look for something that you can address with less pain that this one 😉 But first, I think it is better to finish this one. Changelog is missing :)
> 
> > and suggest some technologies that I can learn simultaneously? Currently, I am learning Kubernetes and am confused about whether to learn Rust or Golang.
> 
> Not sure what to say here 😅 I may be a bit biased by my own preferences. However, I have a non-technical suggestion: don't open multiple front lines at the same time. It's not about specializing only in one thing, but to avoid covering too much at once and having a rough understanding of nothing. IMHO, it is better to invest the required time in one technology to understand it well enough for being able to deliver good quality work. Particularly at the beginning, to lay a solid foundation.
> 
> I know I didn't give you a path to follow, but that was on purpose 😜 Hope you don't mind 😇

Thank you so much for the suggestion sir, I will practice more of MERN stack and try to improve my understanding.

Reply:
> > Hello sir, can you please suggest to me what to work on next
> 
> Sure, we can look for something that you can address with less pain that this one 😉 But first, I think it is better to finish this one. Changelog is missing :)
> 

Sir, could you please tell me how to create a changelog file. I can't find it in the repository.